packages/beachball/src/help.ts

```diff
@@ -21,13 +21,14 @@ Commands:
   changelog           - based on change files, create changelogs and then unlinks the change files
   bump                - bumps versions as well as generating changelogs
   publish             - bumps, publishes to npm registry (optionally does dist-tags), and pushes changelogs back into master
+  sync                - synchronizes published versions of packages from a registry, makes local package.json changes to match what is published
```

Comment:
should also add to:
https://github.com/microsoft/beachball/blob/master/README.md
https://github.com/microsoft/beachball/blob/master/packages/beachball/README.md

Reply:
good point! added!